palantir-java-format/src/main/java/com/palantir/javaformat/doc/BreakTag.java

```diff
@@ -0,0 +1,7 @@
+package com.palantir.javaformat.doc;
+
+import com.google.errorprone.annotations.Immutable;
+
+/** Unique identifier for a break. */
```

Comment:
could we @link to ConditionalBlankLine etc to emphasise why we'd care about having a "unique identifier for a break"?